wpilibc/athena/include/DriverStation.h

```diff
@@ -68,6 +68,7 @@ class DriverStation : public SensorBase, public RobotStateInterface {
   Alliance GetAlliance() const;
   int GetLocation() const;
   void WaitForData();
+  bool WaitForData(long long timeout);
```

Comment:
Ok I changed this. I also added the note to the doc we have that this is changing.